### PR TITLE
shellcheck results

### PR DESCRIPTION
**Pull request by whytewolf**
shellcheck results. minor changes around the formatting. no functionality change. although it should now be able to handle spaces. 

fix for https://github.com/saltstack/salt/issues/63517

### CODE REVIEW

**Review by dmurphy18 — APPROVED** [review]

**Comment by dwoz** [comment]
I think we also need to add a test that runs shellcheck on scripts created by relenv.

**Comment by dwoz** [comment]
Should fix #89

**Review by cmcmarrow — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Review by MKLeb — APPROVED** [review]

File: .github/workflows/test-action.yml [hunk 1]
```diff
@@ -35,6 +35,16 @@ jobs:
         run: |
           pip3 install nox
 
+      - name: Install Linux dependencies
```
Comment (s0undt3ch) [inline .github/workflows/test-action.yml:38]: https://github.com/saltstack/salt/blob/master/.github/actions/setup-shellcheck/action.yml ?

**Review by s0undt3ch — APPROVED** [review]

File: .github/workflows/test-action.yml (hunk 1 above)
Comment (dwoz) [reply .github/workflows/test-action.yml:38]: That'd be great to have someplace where we can re-use it, for now I'll proceed.

**Pull request merged**